modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -125,6 +125,65 @@ definition {
 		takeScreenshot();
 	}
 
+	@description = "LPS-142322. Verifies side panel displays information about a violation rule."
+	@priority = "4"
+	test CanDisplayViolationRuleDetails {
+		property osgi.module.configuration.file.names = "com.liferay.frontend.js.a11y.web.internal.configuration.FFA11yConfiguration.config";
+		property osgi.module.configurations = "enable=B&quot;true&quot;";
+
+		// Workaround for side panel sometimes not displaying LPS-141442
+
+		while (IsElementNotPresent(locator1 = "A11yTool#VIOLATION_PANEL_HEADER")) {
+			Refresh();
+		}
+
+		task ("Verify and click on a rule listed on the side panel") {
+			VerifyElementPresent(
+				key_text = "button-name",
+				locator1 = "A11yTool#VIOLATION_PANEL_RULE_LIST_ITEM");
+
+			Click(
+				key_text = "button-name",
+				locator1 = "A11yTool#VIOLATION_PANEL_RULE_LIST_ITEM");
```

Comment:
What is `button-name` referring to? Is there an option to have the text be something like `button name` instead?

What's the difference between `A11yTool#VIOLATION_PANEL_RULE_LIST_ITEM` and `A11yTool#VIOLATION_PANEL_COMPONENT_TITLE`? Since they use the same key variable?

Reply:
button-name is the name of the violation 
![image](https://user-images.githubusercontent.com/28836926/142079034-4f09d02d-2489-4eef-847e-ba468c167502.png)

Reply:
`A11yTool#VIOLATION_PANEL_RULE_LIST_ITEM ` is each item listed in the top-level view, shown in the screenshot above
`A11yTool#VIOLATION_PANEL_COMPONENT_TITLE` is the title in the next view when you click on the rule 
![image](https://user-images.githubusercontent.com/28836926/142079262-7744d95a-385b-4f03-bcc7-9edd586b9e42.png)

Reply:
i called it component title, cause if you click on an Occurrence, the next view "Occurence 1" title shares the same path 
![image](https://user-images.githubusercontent.com/28836926/142079369-28e19deb-2661-4c22-9056-25e159b928cc.png)

Reply:
makes sense, thanks for explaining @susanchen3!